CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 
 ### :rocket: (Enhancement)
 
+* feat: print instrumentation library info with console span exporter [#4848](https://github.com/open-telemetry/opentelemetry-js/pull/4848) @blumamir
```

Comment:
Definitely doesn't have to be part of this PR, but could consider doing the same for https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/sdk-logs/src/export/ConsoleLogRecordExporter.ts

I'd have said for https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/sdk-metrics/src/export/ConsoleMetricExporter.ts#L74 as well, but it currently doesn't even include the `resource` yet.

Reply:
Added the instrumentation scope also in log record exporter.
For metrics, I prefer not to address as part of this PR.